front/app/containers/Admin/projects/project/analysis/Heatmap/index.tsx

```diff
@@ -69,13 +70,20 @@ const Heatmap = () => {
   if (
     !userCustomFields ||
     !filteredInputCustomFields ||
-    !statisticalInsightsEnabled ||
     !project ||
     authUser?.data.attributes.highest_role !== 'super_admin'
   ) {
     return null;
   }
 
+  if (!autoInsightsAllowed) {
+    return (
+      <Tooltip content={formatMessage(messages.autoInsightsUpsellNudge)}>
```

Comment:
@IvaKop We have a reusable UpsellTooltip in the components folder with exactly this message. :)

Reply:
Ah, I didn't know. Thanks, will change it :)